eventio/tests/test_low_level.py

```diff
@@ -0,0 +1,34 @@
+import pkg_resources
+import eventio
+import struct
+from os import path
+
+one_shower = pkg_resources.resource_filename(
+    'eventio', path.join('resources', 'one_shower.dat')
+)
+
+three_with_reuse = pkg_resources.resource_filename(
+    'eventio', path.join('resources', '3_gammas_reuse_5.dat')
+)
+
+import eventio.object_header as oh
```

Comment:
I think the imports should go into the test functions, so a failed import only affects the test where the import is needed